Intersect.Server/Database/Item.cs

```diff
@@ -85,6 +67,17 @@ public string StatBuffsJson
         [NotMapped]
         public int[] StatBuffs { get; set; } = new int[(int) Stats.StatCount];
 
+        [NotMapped]
+        public ItemProperties ItemProperties { get; set; }
```

Comment:
We're already in the Item class, the property should just be named `Properties`.